Sources/Testing/Test.swift

```diff
@@ -210,17 +208,48 @@ extension Test {
     /// - ``Test/parameters``
     public var parameters: [Parameter]?
 
+    /// Serializable representation of the comments associated with this test.
```

Comment:
Perhaps just use the documentation comments from the original symbols? "Serialized representation" isn't technically correct.